preprocessing/nextclade/src/loculus_preprocessing/processing_functions.py

```diff
@@ -196,23 +200,6 @@ def process_date(
         warnings = []
         errors = []
 
-        # TODO: required check is also in process_single - check if can be removed here
```

Comment:
I confirmed with the `missing_one_required_field` test that this error will still be raised